pages_builder/pages/external-link.yml

```diff
@@ -4,3 +4,4 @@ examples:
   -
     link: https://www.example.com'
     text: Example
+    newwindow: true
```

Comment:
I'm not so familar with how we usually do things on the toolkit but one option would be to instead pass in a `target` attribute which is more flexible then just one option (`_blank). However, using `newwindow` abstracts away from the HTML which might be what we want with the toolkit? Not sure if either way is preferred but may be good to think about if you haven't already.

Oh and if we keep it as `newwindow` then it looks like convention is to use camel case (`newWindow`).